jobsdb/jobsdb.go

```diff
@@ -2215,6 +2307,7 @@ func (jd *Handle) getJobsDS(ctx context.Context, ds dataSetT, lastDS bool, param
 	resultsetStates := map[string]struct{}{}
 	for rows.Next() {
 		var job JobT
+		var payload Payload
```

Comment:
Why couldn't this be just a []byte?

Reply:
`rows.Scan` errors if we scan one type into the other.

Reply:
What about `jsParameters`? it's a jsonb. but we're reading []byte. I think this should work. I myself wrote a test and it seems to be working fine with jsonb as well as text. Could you double check.

Reply:
If the payload column were to be `TEXT` then we'd see an error when scanning into `[]byte`

Reply:
Correction: `[]byte` satisfies all of them. will revert this bit